lib/controllers/AssetController.ts

```diff
@@ -76,6 +91,43 @@ export class AssetController extends CRUDController {
     return { measures };
   }
 
+  async unlinkCategory (request: KuzzleRequest) {
```

Comment:
- Done
 - Done
 - I intensivly use updateRequest so I think it's more readable to have intermediary variable instead of writing request.input.body.category many times.